modules/bazel_skylib_gazelle_plugin/1.4.1/source.json

```diff
@@ -0,0 +1,4 @@
+{
+    "integrity": "sha256-CkZrYfMxWF8G7Nu/JIC57fcOBnpT8mHgWWrNVzp9LcM=",
+    "url": "https://github.com/bazelbuild/bazel-skylib/releases/download/1.4.1/bazel-skylib-gazelle-plugin-1.4.1.tar.gz"
```

Comment:
note: it's too bad we called this "plugin" since gazelle uses the term "extension" :(